scripts/gdb-pretty-printers.py

```diff
@@ -0,0 +1,151 @@
+# Copyright (c) 2022, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import gdb
+
+
+if not 'ThrustVectorPrinter' in locals():
+    raise Exception("This file expects the Thrust pretty-printers to be loaded already. "
+                    "Either load them manually, or use the generated load-pretty-printers "
+                    "script in the build directory")
+
+
+class HostIterator():
```

Comment:
Parentheses after class names are not required. This form with empty parentheses was once necessary, but that predates my knowledge of the language. https://docs.python.org/3/tutorial/classes.html#class-definition-syntax
```suggestion
class HostIterator:
```

Reply:
What can you say, I'm old school 😄
<sub>or maybe just haven't worked on a decently sized Python project before?</sub>